client/src/views/client-area/statistics/CustomOrderStatisticsPage.vue

```diff
@@ -0,0 +1,719 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+<template>
+  <div class="client-page-statistics">
+    <template v-if="organizationStats && contractDetails">
+      <!-- active users -->
+      <div class="users active">
+        <div class="users-text">
+          <ion-title class="users-active-text__title title-h2">
+            {{ $msTranslate('clientArea.statistics.titles.users') }}
+            <span class="subtitles-normal">
+              {{ $msTranslate('clientArea.statistics.titles.active') }}
+            </span>
+          </ion-title>
+        </div>
+        <div class="users-cards-list">
+          <ion-card class="users-cards-list-item">
+            <div class="users-cards-list-item-icons">
+              <ion-icon
+                class="users-cards-list-item-icons__main"
+                :icon="person"
+              />
+              <ion-icon
+                class="users-cards-list-item-icons__secondary"
+                :icon="star"
+              />
+            </div>
+            <div class="users-cards-list-item-text">
+              <ion-text class="users-cards-list-item-text__number title-h1">{{ organizationStats.adminUsersDetail }}</ion-text>
+              <ion-text class="users-cards-list-item-text__text subtitles-sm">
+                {{
+                  $msTranslate(
+                    organizationStats.adminUsersDetail.active > 1 ? 'clientArea.statistics.admins' : 'clientArea.statistics.admin',
```

Comment:
Use the plural translation. You can define in the translation file:
`"admin": "Administrator | Administrators"`
and then use
`$msTranslate({ key: 'clientArea.statistics.admin', count: organizationStats.adminUsersDetail.active })`
and it will automatically choose whether to display `Administrator` or `Administrators` based on `count`.